### DISCUSSION TITLE
v3.2.8

### DISCUSSION THREAD

**Discussion by charabaruk** [PodcastGenerator/PodcastGenerator#719] [Announcements]
# Podcast Generator 3.2.8

Podcast Generator 3.2.8 is now available:

<https://github.com/PodcastGenerator/PodcastGenerator/releases/download/v3.2.8/PodcastGenerator-v3.2.8.zip>

This is a patch release for 3.2 containing fixes for episode uploads and the setup procedure.

Support for the 3.2.x version series will end one month after release of Podcast Generator 3.3 or any newer non-bugfix version.

## Prerequisites

* PHP 7.0 or newer (8.0 or newer recommended)
* PHP fileinfo extension
* PHP gettext extension
* PHP simplexml and libxml extensions
* Apache 2 or nginx web server

## Installation

See the `INSTALL` or `INSTALL.nginx` files in the root of the zip file for details on how to perform a fresh install of Podcast Generator for Apache 2 or nginx web servers.

### Upgrading from Previous Versions

Upgrading is supported from any release of Podcast Generator 3.1. Directly upgrading from older versions is not supported: If you are running PG 3.0 then you need to first upgrade to 3.1; if you are running PG 2.7 or older you need to upgrade to 3.0 and then to 3.1 before upgrading to PG 3.2.

**Be sure to make a backup of your site before upgrading!**

1. Download `PodcastGenerator-v3.2.8.zip` and extract to a local directory.
2. Upload (or copy) the content of the `PodcastGenerator/` directory to the appropriate location on your web server.
3. Adjust the permissions for the PHP files to `775` if necessary.
4. Open your website in a web browser to kick off the automated upgrade process.
5. Delete the `components/getid3/` directory from your website installation (the getid3 library has been moved to the new `vendor/` directory).

For safety reasons, the automated upgrade process does not remove any existing files, which is why you need to be able to delete `components/getid3/` yourself.

## Changes

See the changelog for a non-exclusive list of changes since the 3.1.x series.

## Contributors

Thanks to pbassham for the FTP upload fix. Also, thanks to everyone who reported bugs and to our translators.

<hr /><em>This discussion was created from the release <a href='https://github.com/PodcastGenerator/PodcastGenerator/releases/tag/v3.2.8'>v3.2.8</a>.</em>

**Comment by AlexEneas** [comment]
I've upgraded and now trying to get to the podcastgen page on my server, and it's coming up with a HTTP error 500. My actual website is working, just not the podcastgen pages...

**Reply by charabaruk** [reply]
Which particular pages? Do you have a PHP error log that I could review? Please open an issue with details about the problem.
